py/selenium/webdriver/remote/webelement.py

```diff
@@ -47,7 +47,25 @@ class WebElement(object):
     reference is still valid.  This essentially determines whether or not the
     element is still attached to the DOM.  If this test fails, then an
     ``StaleElementReferenceException`` is thrown, and all future calls to this
-    instance will fail."""
+    instance will fail.
+
+    If an element is no longer part of the DOM a StaleElementReferenceException
```

Comment:
again this is not really an appropriate place to be putting this sort of documentation